koku/api/query_handler.py

```diff
@@ -48,24 +50,21 @@ def convert_value(self, value, expression, connection):
         return value.strftime('%Y-%m-%d')
 
 
-class QueryHandler(object):
+class QueryHandler:
     """Handles report queries and responses."""
 
-    def __init__(self, query_parameters, url_data,
+    def __init__(self, query_parameters,
```

Comment:
The `url_data` parameter got removed from the QueryHandler class because nothing references it. It is still present in the provider-specific classes where it is being used. 

This is a good lead-in to the proposed initialization refactoring, but in terms of this PR it's done purely because pylint was complaining about the unused parameter.